src/sentry/static/sentry/app/views/organizationProjectsDashboard/resources.jsx

```diff
@@ -9,6 +9,10 @@ import ErrorRobot from 'app/components/errorRobot';
 import {Panel} from 'app/components/panels';
 import {t} from 'app/locale';
 
+import releasesImg from '../../../images/releases.svg';
```

Comment:
we can probably add an alias for `images/` so these become `images/releases.svg` etc.

Reply:
Yeah we should. I'm not worried about it right now though, we can clean this up later